curriculum/challenges/english/14-responsive-web-design-22/learn-css-flexbox-by-building-a-photo-gallery/61537c9eecea6a335db6da79.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-5
 
 # --description--
 
-Below your `header`, create a new `div` element and assign a class of `gallery` to it.
+Below your `header`, create a new `div` element and assign a class of `gallery` to it. The class of 'gallery' is an identifer that will be referenced to style the image gallery and make it responsive.
```

Comment:
I think simpler is better:
```suggestion
Below your `header` element, create a new `div` element with the `id` of `"gallery"`. This `div` will act as a container for the gallery images.
```

Reply:
I am worried about `"gallery"`, people may type `<div id=""gallery"">`, what do you think of removing the quotes? @jeremyIt

Reply:
We have done ""s around string literals in other challenges. We should pick one and stick to it.